src/main/java/se/nbis/sftp-squid/PasswordAuth.java

```diff
@@ -0,0 +1,44 @@
+package se.nbis.sftpsquid;
```

Comment:
Yes, all classes need to live in a namespace, the Java Way(tm) is to reverse the domain name of the organisation as the initial namespace.